frontend/packages/console-shared/src/sorts/pvc.ts

```diff
@@ -0,0 +1,5 @@
+import * as UIActions from '@console/internal/actions/ui';
+import { K8sResourceCommon } from '@console/internal/module/k8s';
+
+export const pvcUsed = (pvc: K8sResourceCommon): number =>
```

Comment:
I feel like this is an unnecessary abstraction. We can just use the redux `useSelector` hook where we need to access this value.

Reply:
So `pvcUsed` is being passed to Table Component and using hook with a class component is not possible.

Reply:
Okay, that makes sense. Maybe just use a more descriptive name here, like `getPVCUsedCapacity`.

Reply:
I chose this name since the sort naming convention does not use `get`. PTAL https://github.com/openshift/console/blob/master/frontend/public/components/factory/table.tsx#L127-L133